### ISSUE TITLE
Search Page - 1.4.3 Contrast - Contrast errors on search suggestions

### ISSUE THREAD

**Issue by phudson-he** [archesproject/arches#12617]
_**Note:** This issue was found using arches 7.6.x - but I will be guided as to what branch the feature branch should be based on and what target branch for the PR merging etc. This issue is also present on any select2 multi-select/choice fields._

**Steps to Reproduce:**

- Given I am an unregistered or registered user viewing the Search Page
- And I have vision issues
- When I enter a partial search term into the search bar
- I am presented with the dropdown of suggested results
- I then hover over and select an option
- Then I am able to clearly see the text of the suggested search on hover, and the text of the selected search term in the search field.

**Expected Result:**
The user is able to read the white text on the blue background of any highlighted search suggested and any selected search suggestion. This fails [WCAG accessibility guideline 1.4.3](https://www.wcag.com/designers/1-4-3-color-contrast/)

**Actual Result:**
Both the highlighted search suggestion and the selection search suggestion colour ratios are flagged by accessibility checker software as having low contrast. If accessibility mode it turned on then the colour contrast should meet the requirements.

**Evidence:**

<img width="509" height="316" alt="Image" src="https://github.com/user-attachments/assets/0b15eb40-54cd-4034-b29a-61100c60d96c" />

**Comment by phudson-he** [comment]
Closing as fixed/merged here: #12656
